servicematrix/sm-discontinued.include.md

```diff
@@ -0,0 +1,3 @@
+## ServiceMatrix is no longer available. For more details, [please read this message](http://www.particular.net/servicematrix)
```

Comment:
Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not. Introductory phrases and clauses usually begin with a subordinate conjunction (such as after, because, though, or when).

I now use a tool to check my often misuse of commas. The usage of comma in this instance is correct.